packages/eslint-config-base/index.js

```diff
@@ -131,7 +131,7 @@ module.exports =  {
     ],
     'max-depth': ['warn', 3],
     'max-lines-per-function': ['warn', {
-      max: 50,
+      max: 75,
```

Comment:
I see that it was mentioned that our screens can show more than 50 lines. But the idea is that we do not create functions that are too big. I don't think that this a reasonable argument. Best practices state that a function should be max 30 lines long.